qiskit/providers/ibmq/job/exceptions.py

```diff
@@ -42,3 +42,7 @@ class IBMQJobInvalidStateError(IBMQJobError):
 class IBMQJobTimeoutError(JobTimeoutError, IBMQJobError):
     """Errors raised when a job operation times out."""
     pass
+
+
+class IBMQJobUpdateError(IBMQJobError):
```

Comment:
I believed adding the exception was a good idea, in order to specify the type of error that occurred. But, it is likely better to use `IBMQJobApiError` since it's relevant and doesn't require a new exception class 👍